stackdriver_test.go

```diff
@@ -136,3 +137,23 @@ func TestGRPC(t *testing.T) {
 
 	client.Single(context.Background(), &testpb.FooRequest{SleepNanos: int64(42 * time.Millisecond)})
 }
+
+func TestUserAgent(t *testing.T) {
+	e, err := NewExporter(Options{UserAgent: "test"})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if want, got := "test", e.statsExporter.o.UserAgent; want != got {
+		t.Fatalf("UserAgent = %q; want %q", got, want)
+	}
+
+	e, err = NewExporter(Options{UserAgent: "test {{sdk-version}}{{exporter-version}}"})
```

Comment:
Shall we add a real world example here (e.g. OpenTelemetry Collector) to make it easier to see what's the use case?

Reply:
I put OpenCensus Service instead 😄